lib/net/java/dev/spellcast/utilities/SortedListModel.java

```diff
@@ -139,6 +139,7 @@ public int indexOf( final Object o )
 	@Override
 	public boolean contains( final Object o )
 	{
+		//noinspection ListIndexOfReplaceableByContains
```

Comment:
not really keen on these IDE specific comments

Reply:
Me either. But the suggested refactoring caused an infinite loop, if adopted.

I wanted to never see it again.
I can remove it.

Reply:
If most of us are using Intellij, you can leave it. Normally I just avoid doing anything in the case that the suggestion is wrong.

There are more than just this one -- search for "//noinspection" to find them.